scripts/pack-packages.sh

```diff
@@ -0,0 +1,30 @@
+# This script is used in the "npm pack" step in our CI build pipelines.
+# It runs (p)npm pack for all packages and sorts them into scoped/unscoped folders.
+# It also outputs a packagePublishOrder.txt file that contains the order that the packages should be published in.
+
+echo PACKAGE_MANAGER=$PACKAGE_MANAGER
+echo PUBLISH_NON_SCOPED=$PUBLISH_NON_SCOPED
+echo RELEASE_GROUP=$RELEASE_GROUP
+echo STAGING_PATH=$STAGING_PATH
+
+mkdir $STAGING_PATH/pack/
+mkdir $STAGING_PATH/pack/scoped/
+mkdir $STAGING_PATH/test-files/
+
+if [[ "$PUBLISH_NON_SCOPED" == "True" ]]; then
+  mkdir $STAGING_PATH/pack/non-scoped/
+fi
+
+if [ -f ".releaseGroup" ]; then
+  flub exec --no-private --concurrency=1 --releaseGroup $RELEASE_GROUP -- "$PACKAGE_MANAGER pack"
+
+  flub exec --no-private --concurrency=1 --releaseGroup $RELEASE_GROUP -- "mv -t $STAGING_PATH/pack/scoped/ ./*.tgz"
+
+  flub exec --no-private --releaseGroup $RELEASE_GROUP -- "[ ! -f ./*test-files.tar ] || (echo 'test files found' && mv -t $STAGING_PATH/test-files/ ./*test-files.tar)"
```

Comment:
I think removing the `&&` in between the commands will make it so the next ones still execute here even if the previous ones fail?

Reply:
I believe [Craig's suggestion](https://github.com/microsoft/FluidFramework/pull/16161/files#r1297637978) takes care of this, resolving.

Reply:
Fixed.